code/game/objects/random/biomes/biome.dm

```diff
@@ -54,7 +54,7 @@ GLOBAL_LIST_EMPTY(loot_biomes)
 	var/danger_level = 0
 	var/only_top = 1
 	var/range = 7
-	var/can_burrow = FALSE
+	var/can_burrow = TRUE
```

Comment:
???

Reply:
this change does nothing, it should be TRUE by default, because the documentation, but it does not have any real changes in the game. I have changed it because everything that is in this pr went together with some changes to the map but I moved a lot of the changes to this pr so that the maintainers could review it more easily, however this comes from there and I repeat, it does not affect the game at all except now when mapping this will be the default value.